include/groonga/groonga.h

```diff
@@ -219,13 +219,32 @@ grn_log_level_parse(const char *string, grn_log_level *level);
    GRN_QUERY_LOG_SCORE)
 #define GRN_QUERY_LOG_DEFAULT GRN_QUERY_LOG_ALL
 
+/** \brief The input/output content format. */
 typedef enum {
+  /**
+   * The content is in no specific format. This means that either no content is
+   * produced or the content remains in its original format.
+   */
   GRN_CONTENT_NONE = 0,
+  /** The content is in a tab-separated values (TSV) format. */
   GRN_CONTENT_TSV,
+  /** The content is in JSON format. */
   GRN_CONTENT_JSON,
+  /** The content is in XML format. */
   GRN_CONTENT_XML,
+  /**
+   * The content is in MessagePack format. This requires that MessagePack is
+   * available during Groonga's build process. Without it, this type cannot be
+   * used.
+   */
   GRN_CONTENT_MSGPACK,
+  /** The content is a list of Groonga commands. */
   GRN_CONTENT_GROONGA_COMMAND_LIST,
+  /**
+   * The content is in Apache Arrow format. This requires that Apache Arrow is
```

Comment:
Apache Arrow has 2 IPC formats:
* IPC Streaming Format: https://arrow.apache.org/docs/format/Columnar.html#ipc-streaming-format
* IPC File Format: https://arrow.apache.org/docs/format/Columnar.html#ipc-file-format

We use only the streaming format: https://groonga.org/docs/reference/commands/load.html#values

(Oh... https://groonga.org/docs/reference/command/output_format.html doesn't mention the Apache Arrow format output...)

Reply:
fix: [619364e](https://github.com/groonga/groonga/pull/2255/commits/619364ebcf065f427053c207624ad9fa25929dc7) Thanks. I will update the `output_format` documentation in another PR.